src/lib/y2storage/device.rb

```diff
@@ -99,12 +150,12 @@ def siblings
 
     # Checks whether the device is a concrete kind(s) of device.
     #
-    # Always false for this base class, which represents an abstract device.
-    # To be redefined by classes representing more concrete devices.
+    # @note Always false for this base class, which represents an abstract device.
```

Comment:
I would not say this is a note. For me, all the text is an integral part of the description of the method. IMHO `@note` is more for small annotations about unexpected behaviors and so on.